package.json

```diff
@@ -1,11 +1,10 @@
 {
```

Comment:
I think it worth to add "engines" section and mention, that this package works only with node >= xxx (Maybe in node with generators??)